ui/lib/pki/addon/components/pki-configure-form.hbs

```diff
@@ -17,8 +17,8 @@
               id={{option.key}}
               name="pki-config-type"
               @value={{option.key}}
-              @groupValue={{this.actionType}}
-              @onChange={{fn (mut this.actionType) option.key}}
+              @groupValue={{@config.actionType}}
```

Comment:
Since we're swapping this out for an attribute on the model let's go ahead and remove the tracked property from the component class

Reply:
Ahh yes good catch on that!